migrations/00064_add_redirects.sql

```diff
@@ -0,0 +1,158 @@
+-- +goose Up
+/**********************************************************/
+/*** SCRIPT AUTHOR: Matjaz Debelak (matjaz@debelak.org) ***/
+/***    CREATED ON: 2022-11-17T10:39:12.129Z            ***/
+/**********************************************************/
+
+--- BEGIN CREATE TABLE "public"."redirects" ---
+
+CREATE TABLE IF NOT EXISTS "public"."redirects" (
+	"id" uuid NOT NULL  ,
+	"status" varchar(255) NOT NULL DEFAULT 'draft'::character varying ,
+	"user_created" uuid NOT NULL  ,
+	"date_created" timestamptz NULL  ,
+	"user_updated" uuid NOT NULL  ,
+	"date_updated" timestamptz NULL  ,
+	"target_url" varchar(255) NOT NULL  ,
+	"code" varchar(255) NOT NULL DEFAULT NULL::character varying ,
+	CONSTRAINT "redirects_pkey" PRIMARY KEY (id) ,
+	CONSTRAINT "redirects_user_updated_foreign" FOREIGN KEY (user_updated) REFERENCES directus_users(id) ,
+	CONSTRAINT "redirects_user_created_foreign" FOREIGN KEY (user_created) REFERENCES directus_users(id)
+);
+
+ALTER TABLE IF EXISTS "public"."redirects" OWNER TO builder;
+
+GRANT SELECT ON TABLE "public"."redirects" TO api; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT INSERT, INSERT, UPDATE, DELETE ON TABLE "public"."redirects" TO directus; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
```

Comment:
Duplicate insert grant? It probably doesn't fail but it's marked as an error here.